Task 3/Task3.2/DynamicArray.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Task3._2
+{
+    class DynamicArray<T>:ICollection<T>,IEnumerable<T>
+    {
+        public int index = -1;
+        private int capacity = 8;
+        private int length = 0;
+        private T[] data;
+        public DynamicArray()
+        {
+            Array.Resize(ref data, Capacity);
+        }
```

Comment:
```suggestion
        private T[] data;
        
        public DynamicArray()
        {
            Array.Resize(ref data, Capacity);
        }
```
Не забывай разделять члены класса пустой строкой. И еще не увидел конструктор с целочисленным параметром.

Reply:
Сделал этот конструктор.